tools/refguide_check.py

```diff
@@ -561,11 +561,13 @@ def main(argv):
         args.module_names = list(PUBLIC_SUBMODULES) + [BASE_MODULE]
 
     module_names = list(args.module_names)
-    for name in module_names:
-        if name in OTHER_MODULE_DOCS:
-            name = OTHER_MODULE_DOCS[name]
-            if name not in module_names:
-                module_names.append(name)
+    module_names += [
```

Comment:
unless I'm missing something, the line above could be removed by doing something like

```suggestion
    module_names = args.module_names + [
```